app/main.js

```diff
@@ -234,15 +238,15 @@ function pauseBreaks (seconds) {
     finishBreak()
   }
   breakPlanner.pause()
-  if (seconds !== 1) {
-    resumeBreaksTimer = setTimeout(resumeBreaks, seconds)
+  if (milliseconds !== 1) {
+    resumeBreaksScheduler = new Scheduler(resumeBreaks, milliseconds)
```

Comment:
I would try to get `milliseconds` to `breakPlanner.pause()` as parameter and get rid of need to have `Scheduler` in `main.js` 

What do you  think about it?